Kernel/Jail.h

```diff
@@ -52,7 +56,9 @@ class Jail : public AtomicRefCounted<Jail> {
     using List = IntrusiveListRelaxedConst<&Jail::m_list_node>;
 
 private:
-    NonnullRefPtr<ProcessList> const m_process_list;
+    // NOTE: Although we access this RefPtr during runtime multiple times, we never set
+    // this pointer after construction so it's safe to leave this without any kind of locking.
```

Comment:
This comment seems unrelated and maybe a bit too much since the `RefPtr` is already `const`?